specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure IoT Central",
+    "description": "Azure IoT Central is a service that makes it easy to connect, monitor, and manage your IoT devices at scale.",
+    "version": "1.0.0"
```

Comment:
I thought API versions were supposed to be date-based, like 2021-04-16. Has something changed @johanste ?

Reply:
IoT Central uses path based versioning with /preview and 1.0 and not date based versioning. Both are supported based on the Microsoft REST guidelines and we reviewed this with the API review board.

Reply:
Johnson referred me to [this](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#12-versioning) document. I was unaware this was an option. Thanks!